libretro_core_options.h

```diff
@@ -1008,7 +1008,7 @@ struct retro_core_option_v2_definition option_defs_us[] = {
       BEETLE_OPT(skip_bios),
       "Skip BIOS",
       NULL,
-      "Skips the PlayStation BIOS boot animation normally displayed when loading content. Note: Enabling this causes compatibility issues with a number of games (PAL copy protected games, Saga Frontier, etc.).",
+      "Skip the PlayStation BIOS boot animation normally displayed when loading content. Note: Enabling this causes compatibility issues with a number of games (PAL copy protected games, Saga Frontier, etc.).",
```

Comment:
"when loading content" lacks the "a" article. Would that be enough to make it better in English? I kinda have the feeling that something is still off there.

Reply:
> "when loading content" lacks the "a" article.

Once again, this is the general case. Let me explain. The full sentence is actually:
"Skip the PlayStation BIOS boot animation normally displayed when loading _any_ content"

Any can be omitted in some cases, such as above, but it cannot be replaced by an article.

Reply:
The best I can offer you rn is this:
Skip the PlayStation BIOS boot animation which is normally displayed when loading content.